README.md

```diff
@@ -1 +1,54 @@
-# microverse
\ No newline at end of file
+# Hello-Microverse
+
+## "Hello Microverse" project
+
+the purpose of this project is for mastering best practise, familiar with all the tools used such as linters, github, html and css
+
```

Comment:
- [ ] To make your README look more professional, you should consider adding a screenshot of your project here.

Reply:
Considering this is my first PullRequest experience, is better to overlook some unnecessary things, Rome was not built in a day and this is not encouraging at all compared to my partner PR.